static/src/javascripts/projects/common/views/commercial/survey/survey-adblock.html

```diff
@@ -24,33 +29,53 @@ <h3 class="survey-text__header">
         </h3>
         <div class="survey-content">
             <div class="survey-text">
-                <p class="survey-text__info">
+                <p class="survey-text__strong">
                     <%=surveyText%>
                 </p>
                 <% if (surveyTextSecond) { %>
-                <p class="survey-text__info">
-                    <%=surveyTextSecond%>
-                </p>
+                    <p class="survey-text__strong">
+                        <%=surveyTextSecond%>
+                    </p>
                 <% } %>
                 <% if (surveyTextThird) { %>
                 <p class="survey-text__info">
                     <%=surveyTextThird%>
                 </p>
                 <% } %>
+                <% if (membershipLink || signupLink) { %>
```

Comment:
Oh that was sth that used to be in `frontend.properties` but as we got rid of that I'm not sure it still is possible. A failure should occur at runtime, if there is a failure, which I doubt.